### ISSUE TITLE
Show recent unread news items on homepage

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#3325]
I have the impression that many people are not noticing the news items. Perhaps we should show the most recent three unread news items at the top of the home page (that is, below the navbar, but above everything else)?

**Comment by eliotwrobson** [comment]
Is this still needed? I think we're leaning in the direction of removing news items entirely and converting to github discussions / other methods of announcements.

**Comment by nwalters512** [comment]
Yeah, I don't think we're investing any more in the news items system. We'll continue to use GitHub Discussions for day-to-day deployment announcements. Eventually, I think we'd like to use a blog-like format on our marketing site to announce changes, the main benefit of that being that they can be viewed publicly and aren't tied to a particular server (Canada vs. US. vs...).

GitHub maintains a changelog at https://github.blog/changelog/ and shows top items from it on their logged-in homepage (served via https://github.com/dashboard/changelog):

<img width="381" alt="Image" src="https://github.com/user-attachments/assets/815a75bc-c341-419b-acb3-48a0b08cdda0" />

We could, in the long term, consider something similar.
